### ISSUE TITLE
Settings: Add snap turning

### ISSUE THREAD

**Issue by AtomicAntt** [AtomicAntt/Rewind-Tower#88]
This should be given in seated mode, but also it could be an option for those standing in case there's not much room for the player for extra accessibility.

I think it's more immersive and best if it is disabled, so the default should be disabled, but it is an important accessibility feature.
